quarkchain/cluster/shard_state.py

```diff
@@ -538,6 +539,16 @@ def add_tx(self, tx: TypedTransaction, xshard_gas_limit=None):
             evm_tx = self.__validate_tx(
                 tx, evm_state, xshard_gas_limit=xshard_gas_limit
             )
+
+            # Don't add the tx if the gasprice in QKC is too low.
+            # Note that this is not enforced by consensus,
+            # but miners will likely discard the tx if the gasprice is too low.
+            genesis_gasprice = get_genesis_gasprice(
```

Comment:
nit: rename to `default_token_gasprice` and `get_default_token_gasprice()`??

Reply:
Looks like we use genesis_token and default_token interchangeably although genesis_token is global-wise while default_token is chain-wise.  So the question is which variable should we stick to?

Reply:
good call. I'm seeing inconsistent usages, such as `apply_transaction` is using genesis token while `validate_transaction` is using default chain token. 

the fundamental question is, if we have a new chain with default chain token set to something else (e.g. QZEC / QBTC), the conversion should apply for them, instead of QKC, right? so I would suggest using default chain token and double check existing usage of genesis token is consistent with our expectation